pkg/controller/pxc/controller.go

```diff
@@ -1145,7 +1153,15 @@ func (r *ReconcilePerconaXtraDBCluster) createOrUpdate(ctx context.Context, cr *
 			obj.SetResourceVersion(oldObject.GetResourceVersion())
 		}
 
-		log.V(1).Info("Updating object", "object", obj.GetName(), "kind", obj.GetObjectKind())
+		log.V(1).Info("Updating object",
+			"object", obj.GetName(),
+			"kind", obj.GetObjectKind(),
+			"hashChanged", oldObject.GetAnnotations()["percona.com/last-config-hash"] != hash,
+			"metaChanged", !isObjectMetaEqual(obj, oldObject),
+		)
+		if util.IsLogLevelVerbose() {
+			fmt.Println(cmp.Diff(oldObject, obj))
```

Comment:
I think we should use a `log` here. This can break parsing of logs when json logging format is used.

Reply:
On top of @pooknull comment, I was wondering why we are not used `log.V(2)` which is essentially verbose level 2.